operator/api/v1alpha1/plan_types.go

```diff
@@ -20,27 +20,64 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
-// PlanSpec defines the desired state of Plan
+// PlanSpec defines the Service Plan spec.
 type PlanSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// The name of the Offering this Plan belongs to.
+	Offering string `json:"offering,omitempty"`
```

Comment:
Why is this associated by name rather than ID? Actually what is the ID even used for?

Reply:
Good point. I do by name because I can then get the parent object and set the ownership. This is done using the namespace and name of the object. ID is the unique ID used by the OSBAPI.